gnomad_qc/v4/sample_qc/assign_ancestry.py

```diff
@@ -137,6 +292,7 @@ def prep_ht_for_rf(
     # Use the withhold proportion to create PR curves for when the RF removes
```

Comment:
Im honestly think we should just remove the `withhold_prop` and `max_number_mislabeled_training_samples` and `max_proportion_mislabeled_training_samples` from the code. It's confusing, and we determined it's probably not the right way to do this.